packages/grid_client/scripts/single_vm_zos4.ts

```diff
@@ -0,0 +1,116 @@
+import { Features, FilterOptions, generateRandomHexSeed, GridClient, MachinesDeleteModel, MachinesModel } from "../src";
+import { config, getClient } from "./client_loader";
+import { log, pingNodes } from "./utils";
+
+async function deploy(client, vms) {
+  const resultVM = await client.machines.deploy(vms);
+  log("================= Deploying VM =================");
+  log(resultVM);
+  log("================= Deploying VM =================");
+}
+
+async function getDeployment(client, vms) {
+  const resultVM = await client.machines.getObj(vms.name);
+  log("================= Getting deployment information =================");
+  log(resultVM);
+  log("================= Getting deployment information =================");
+}
+
+async function cancel(client, vms) {
+  const resultVM = await client.machines.delete(vms);
+  log("================= Canceling the deployment =================");
+  log(resultVM);
+  log("================= Canceling the deployment =================");
+}
+
+async function main() {
+  const name = "vm";
+  const grid3 = await getClient(`vm/${name}`);
+  const instanceCapacity = { cru: 2, mru: 4, sru: 100 }; // Update the instance capacity values according to your requirements.
+
+  //VMNode Selection
+  const vmQueryOptions: FilterOptions = {
+    cru: instanceCapacity.cru,
+    mru: instanceCapacity.mru,
+    sru: instanceCapacity.sru,
+    availableFor: grid3.twinId,
+    farmId: 1,
```

Comment:
please remove this from the query as the scripts fails when it can't find a node in farm 1